testkit/tests/counter/main.rs

```diff
@@ -526,6 +526,28 @@ fn test_explorer_blocks_basic() {
     );
 }
 
+#[test]
+fn test_explorer_block_base() {
```

Comment:
IMHO the name of the test doesn't explain what does it check (however, that applies for other tests in that module too...).

Maybe something like `test_explorer_api_block_request`?